fix: migration to bootstrap 5

bodnara opened the pull request:
Renamed the following classes
* left-* to start-*
* right-* to end-*
* ml-* to ms-*
* mr-* to me-*
* pl-* to ps-*
* pr-* to pe-*
* font-weight-* to fw-*

mattjoke commented on libs/perun/dialogs/src/lib/member-tree-view-dialog/member-tree-view-dialog.component.html:
```diff
@@ -2,11 +2,11 @@
   </h1>
-  <mat-spinner *ngIf="loading" class="ml-auto mr-auto"></mat-spinner>
+  <mat-spinner *ngIf="loading" class="ms-auto me-auto"></mat-spinner>
   <div *ngIf="!loading" mat-dialog-content class="mr-0">
```
```suggestion
  <div *ngIf="!loading" mat-dialog-content class="ms-0">
```

mattjoke commented on libs/perun/components/src/lib/attribute-search-select/attribute-search-select.component.html:
```diff
@@ -1,4 +1,4 @@
-<div *ngIf="options !== undefined" class="container ml-0 mr-0 pl-0 pr-0 mt-3">
+<div *ngIf="options !== undefined" class="container ml-0 mr-0 ps-0 pe-0 mt-3">
```
```suggestion
<div *ngIf="options !== undefined" class="container ms-0 me-0 ps-0 pe-0 mt-3">
```

mattjoke reviewed (changes requested):
Just skimmed through the files, however I think someone else needs also to look at this.

HejdaJakub commented on apps/publications/src/app/pages/create-publication-page/import-publications-page/import-publications-page.component.html:
```diff
@@ -22,19 +22,19 @@ <h1 class="page-subtitle d-flex">
       </perun-web-apps-year-range>
 
-      <button mat-stroked-button class="left-space" (click)="searchPublications()">
+      <button mat-stroked-button class="start-space" (click)="searchPublications()">
```
Again incorrect replacement. In this class is more similar cases like that for `start-space` and also for `end-button`.

HejdaJakub commented on apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-settings/vo-settings-application-form/vo-settings-application-form.component.html:
```diff
@@ -78,7 +78,7 @@ <h1 class="page-subtitle">
     <!--      *ngIf="editAuth"-->
     <!--      (click)="clear()"-->
-    <!--      class="ml-2"-->
+    <!--      class="ms-"-->
```
I know it's just a comment, but for consistency.
```suggestion
    <!--      class="ms-2"-->
```

bodnara commented:
- reverted changes to custom classes
- removed decorations on links
- aligned user name and menu in top header
- set proper display of icons inside button (refresh, back, ...)
